### DISCUSSION TITLE
Unable to build the package

### DISCUSSION THREAD

**Discussion by inner214** [wixtoolset/issues#7871] [Questions]
On an attempt to compile the installation package, I'm getting an error saying:

"WIX0200	The Package element contains an unhandled extension element 'WixUI'. Please ensure that the extension for elements in the 'http://wixtoolset.org/schemas/v4/wxs/ui' namespace has been provided."

The code snippet responsible for UI declaration is as follows:

xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui"
...
<ui:WixUI Id="WixUI_Minimal"/>

When I'm trying to open the schema itself, I'm getting the error saying that the page was not found.

Any ideas how to get it solved? Thanks!

**Comment by nirbar** [comment]
Did you add `PackageReference` to the UI extension?

**Comment by inner214** [comment]
Yes, I did. Here is what I have in my wixproj file

  <ItemGroup>
    <PackageReference Include="WixToolset.Heat" Version="4.0.3" />
    <PackageReference Include="WixToolset.Netfx.wixext" Version="4.0.3" />
    <PackageReference Include="WixToolset.UI.wixext" Version="4.0.3" />
  </ItemGroup>

I'm a bit confused the fact I'm unable to open the schema definition itself. Out of curiosity, are you able to open it up following the link below http://wixtoolset.org/schemas/v4/wxs/ui ?

**Reply by inner214** [reply]
Alright, it looks like I've solved the issue. It appears one of the recent Visual Studio updates somehow damaged FireGiant Heatwave installation on my local machine. I've got it re-installed and the solution now builds perfectly well.
